Github/service.js

```diff
@@ -1,134 +1,164 @@
-/***********************************************************
+/* **********************************************************
 // OSTEP Dashboard Github API
 // service.cpp
 // Date Created: 2018/09/22
 // Author: Yiran Zhu, Lewis Kim and Josue Quilon Barrios
 // Email: yzhu132@myseneca.ca
 // Description: Github API that gets all the sorted recent 
 // commits from an organization/user
-***********************************************************/
+********************************************************** */
 
+const request = require('request');
 
-var request = require("request");
-var key = process.env.GITHUB_TOKEN;
-var repoUrl = 'https://api.github.com/orgs/Seneca-CDOT/repos?per_page=100&access_token=' + key;
+const key = process.env.GITHUB_TOKEN;
+const repoUrl = `https://api.github.com/orgs/Seneca-CDOT/repos?per_page=100&access_token=${key}`;
 console.log(key);
-let branchUrls = []
+let branchUrls = [];
 let recentCommits = [];
 
-let today = new Date();
+const today = new Date();
 const recency = 24 * 60 * 60 * 1000;
 
 module.exports.getRepos = () => {
-    repoUrls = [];
-    branchUrls = [];
-    recentCommits = [];
+  branchUrls = [];
+  recentCommits = [];
 
-    return new Promise((resolve, reject) => {
-        request.get({
-            url: repoUrl,
-            headers: { 'User-Agent': 'request' },
-        }, (err, res, data) => {
-            if (err) {
-                console.log('Error:', err);
-                reject("Unable to get repos.");
-            } else {
-                let reposX = JSON.parse(data);
-                reposX.forEach(repo => {
-                    if (new Date(today - new Date(repo.pushed_at)) < new Date(recency)) {
-                        branchUrls.push({
-                            'name': repo.name,
-                            'url': repo.branches_url.slice(0, repo.branches_url.length - 9) + '?per_page=100&access_token=' + key
-                        });
-                    }
-                });
-                resolve();
+  return new Promise((resolve, reject) => {
+    request.get(
+      {
+        url: repoUrl,
+        headers: { 'User-Agent': 'request' },
+      },
+      (err, res, data) => {
```

Comment:
In the future we will need filter out Error Status codes (non 20X) since these bypass if (err) {..} statement.
E.g. I ran into this issue today when github auth token got invalidated somehow and 401 was returned & resulted in ambiguous errors. I think I am going to open a separate issue for this since it's unrelated to this PR.